Cron_Philip/Assignments/dojo_survey_valid/templates/result.html

```diff
@@ -0,0 +1,13 @@
+<html>
+  <head>
+    <title>Result</title>
+  </head>
+  <body>
+    <h1>Submitted Info</h1>
+    <h3>Name: {{session['name']}}</h3>
+    <h3>Dojo Location: {{session['dojo_location']}}</h3>
+    <h3>Favorite language: {{session['favorite_language']}}</h3>
+    <h3>Comment: {{session['comment']}}</h3>
+    <button><a href="http://127.0.0.1:5000/">Go Back</a></button>
```

Comment:
Just use the relative path: href="/" rather than a fully qualified, absolute path. This way, in the future when you deploy to a real server, everything still worx w/o you having to alter your code.